index.html

```diff
@@ -0,0 +1,52 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="/style.css">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.2.1/css/all.min.css">
+    <link href='https://fonts.googleapis.com/css?family=Crete Round' rel='stylesheet'>
+    <link href='https://fonts.googleapis.com/css?family=Inter' rel='stylesheet'>
+    <title>Document</title>
+</head>
+<body>
+    <nav class="nav-bar">
+        <h2 class="nav-name">Mark Fenekayas</h2>
+        <div class="nav-items">
+            <h4 class="items-click">Portfolio</h4>
+            <h4 class="items-click">About</h4>
+            <h4 class="items-click">Contact</h4>
+            <i class="fa-regular fa-envelope items-click items-icon"></i>
+        </div>
+    </nav>
+
+    <section class="first-section">
+        <section class="left-section">
+            <div class="contact-icons">
+                <i class="fa-brands fa-github"></i>
+                <i class="fa-brands fa-linkedin-in"></i>
+                <i class="fa-brands fa-angellist"></i>
+                <i class="fa-brands fa-twitter"></i>
+                <i class="fa-solid fa-m"></i>
+            </div>
+        </section>
+        <section class="central-section">
+            <div class="section-text">
+                <h2>Hey There.<br/>I'm Mark</h2>
+                <h4>I'm a Software Developer</h4>
+                <p>
+                    I can help you build a product, feature or website.
+                    Look through some of my work and experience! If you
+                    like what you see and have a project you need coded, 
+                    don't hesitate to call me.
+                </p>
+            </div>
+        </section>
+        <section class="right-section">
+           <img src="./img/Illustration-header-three.png" alt="Illustration Image" class="image"/>
+        </section>
+    </section>
```

Comment:
- [ ] Kindly remove this image from here and use the CSS background-image property. Using the CSS background-image property is always better if an image is not part of the content. [Check this link](https://stackoverflow.com/questions/492809/when-to-use-img-vs-css-background-image) for more info. :+1: